site2/docs/admin-api-topics.md

```diff
@@ -1666,7 +1666,7 @@ Non-partitioned topics must be explicitly created. When creating a new non-parti
 
 By default, 60 seconds after creation, topics are considered inactive and deleted automatically to avoid generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to a specific value.
 
-For more information about the two parameters, see [here](reference-configuration.md#broker).
+For more information about the two parameters, see [here](reference-configuration-broker.md).
```

Comment:
Hi @alpreu we're generating configuration docs from code automatically, and the new page will be something like https://pulsar.apache.org/tools/pulsar-config/2.11.0-SNAPSHOT/#/

`reference-configuration-broker.md` will be removed from the pulsar repo https://github.com/apache/pulsar/pull/17197

So can you hold on this change?

Feel free to update it after the new page is finalized, thank you!

/cc @SignorMercurio

Reply:
cc @momo-jun @daveduggins

Reply:
@Anonymitaet Note that the new link should be https://pulsar.apache.org/reference

Reply:
@SignorMercurio yes, thanks for reminding me.

Reply:
Hi together, I just went through the current site and updated most occurrences of the reference configuration.
There are a few links left that I could not find an equivalent for, maybe the documentation is outdated for these.
I'm attaching a screenshot below so you can see which ones are still missing:
<img width="954" alt="Screenshot 2022-09-06 at 16 07 37" src="https://user-images.githubusercontent.com/11444089/188659502-6f5d6818-e2d4-4f9c-a697-cca2273f3d8c.png">

Reply:
Thanks! @alpreu 
I suggest holding these changes since this work is still WIP. @SignorMercurio thoughts?

Reply:
Sorry, missed the notification. I agree that we should hold these changes.d